.github/CODEOWNERS

```
@@ -105,3 +105,19 @@ sycl/test-e2e/KernelFusion @victor-eds @Naghasan @sommerlukas
 
 # Matrix tests
 sycl/test-e2e/Matrix @dkhaldi @YuriPlyakhin @yubingex007-a11y
+
+#Native CPU PI 
+sycl/plugins/native_cpu/ @intel/dpcpp-nativecpu-pi-reviewers
+sycl/test/check_device_code/native_cpu/ @intel/dpcpp-nativecpu-pi-reviewers 
+sycl/test/native_cpu/ @intel/dpcpp-nativecpu-pi-reviewers 
```

```suggestion
sycl/**/native_cpu/ @intel/dpcpp-nativecpu-pi-reviewers
```
I think we can simplify to "any directory named `native_cpu` inside sycl directory is owned by the @intel/dpcpp-nativecpu-pi-reviewers team". Right?

---

Yes, I agree